README.md

```diff
@@ -5,7 +5,7 @@
 Python wrapper for the C implementation of the Generic Sensor Format library.
 
 - Free software: MIT license
-- __Notes on licensing__: The bundled `libgsf/libgsf03_08.so` is covered by the [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html) license. A copy of this license is included in the project at `libgsf/libgsf_LICENSE.md`. This does not affect the top-level MIT licensing of the project. However, as required by the libgsf license, the libgsf shared object library used by gsfpy at runtime may be replaced with a different version by setting the `LD_LIBRARY_PATH` environment variable to the directory location of the new library.
+- __Notes on licensing__: The bundled `libgsf/libgsf03_08.so` is covered by the [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html) license. A copy of this license is included in the project at `libgsf/libgsf_LICENSE.md`. This does not affect the top-level MIT licensing of the project. However, as required by the libgsf license, the libgsf shared object library used by gsfpy at runtime may be replaced with a different version by setting the `GSFPY_GSFLIB_PATH` environment variable to the absolute file path of the new library.
```

Comment:
We should try to be consistent about gsflib vs libgsf and I think the environment variable name should also be held to this standard e.g. `GSFPY_GSFLIB_PATH` vs `GSFPY_LIBGSF_PATH`

Reply:
The shared object library is located at `libgsf/libgsf03-08.so`, so have kept with LIBGSF